pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -883,6 +883,22 @@ func TestCreateDynamicMetadataMatcher(t *testing.T) {
 				},
 			}),
 		},
+		{
+			k: attrRequestGroups, v: "test-groups",
+			expect: generateMetadataListMatcher([]string{attrRequestGroups}, &metadata.ListMatcher{
+				MatchPattern: &metadata.ListMatcher_OneOf{
+					OneOf: &metadata.ValueMatcher{
+						MatchPattern: &metadata.ValueMatcher_StringMatch{
+							StringMatch: &metadata.StringMatcher{
+								MatchPattern: &metadata.StringMatcher_Exact{
+									Exact: "test-groups",
+								},
+							},
+						},
+					},
+				},
+			}),
+		},
```

Comment:
Could you also add a test case in `TestConvertRbacRulesToFilterConfig`? That should cover the full path inside this filter.